pkgs/development/beam-modules/erlang-ls/default.nix

```diff
@@ -1,10 +1,11 @@
-{ fetchFromGitHub, fetchHex, rebar3Relx, buildRebar3, rebar3-proper, stdenv, lib }:
+{ fetchFromGitHub, fetchgit, fetchHex, rebar3Relx, buildRebar3, rebar3-proper
+, stdenv, writeScript, lib }:
 let
   version = "0.18.0";
   owner = "erlang-ls";
   repo = "erlang_ls";
   deps = import ./rebar-deps.nix {
-    inherit fetchHex fetchFromGitHub;
+    inherit fetchHex fetchFromGitHub fetchgit;
```

Comment:
small question, did you need fetchgit for testing ?

Reply:
`rebar-deps.nix` (which is generated by the newest version of rebar3_nix) now requires it as an argument